services/venice-controller/src/main/java/com/linkedin/venice/ingestion/control/RealTimeTopicSwitcher.java

```diff
@@ -185,6 +185,63 @@ long getRewindStartTime(
     }
   }
 
+  public void transmitVersionSwapMessage(Store store, int previousVersion, int nextVersion) {
```

Comment:
We might need adding unit test added in `RealTimeTopicSwitcherTest` for this method.